src/main/madara-app.ts

```diff
@@ -252,3 +252,58 @@ export async function getAppSettings(appId: string) {
   }
   return {};
 }
+
+export async function fetchAllRunningApps(window: BrowserWindow) {
+  const containers = await docker.listContainers();
+  const runningApps: { [appId: string]: boolean } = {};
+
+  containers.forEach((container) => {
+    const containerName = container.Names[0].substring(1); // remove the leading "/"
+    const parts = containerName.split('-');
+    const partialAppId = parts.slice(1, 5).join('-');
+    runningApps[partialAppId] = true;
+  });
+
+  const partialAppIds = Object.keys(runningApps);
+
+  const appsFromConfigRunning = APPS_CONFIG.apps.filter((app) =>
+    partialAppIds.some((partialAppId) => app.id.startsWith(partialAppId))
```

Comment:
why can't we directly equate (instead of `startsWith`)

Reply:
This is cause by the container read from the docker is using 4/5 of the id from the one in the appsconfig's id

Reply:
i can use include to simplify the logic actually

Reply:
What exactly do you mean by 4/5? Also, it's usually a good practice to for the person who created the comment to resolve the issue. This way, we ensure that we only resolve once both of us agree on the solution :)

Reply:
so example like the container name is 'tsukuyomi-6430cd1b-097d-46be-a44d-aa7631433910-apibara-starknet
' and the id to that app is 6430cd1b-097d-46be-a44d-aa7631433910 thats why i did a slice on the top to join 1 and 5. Maybe any idea on a better way to tackle this ?

Reply:
Ahh ok, that's not so neat. Instead lets do `containerName.includes(app.id)`